jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/README.md

```diff
@@ -344,5 +353,131 @@
+
+    ```
+## **4. Predicting Entities**
+* For predicting entities we are going execute walker `predict_entity_from_tfm` and provide text input in context and will get output `list of entities` available in text-data.
+Run the following command to execute walker `predict_entity_from_tfm`
+
+    ```
+    walker run predict_entity_from_tfm -ctx "{\"text\":\"It was the second costly blunder by Syria in four minute\"}"
+    ```
+* After executing walker `predict_entity_from_tfm` , will get `predicted entities` list as
+    ```
+    [
+        {
+            "entity_value": "syria",
+            "entity_type": "B-LOC",
+            "score": 0.8613966703414917,
+            "start_index": 36,
+            "end_index": 41
+        }
+    ]
+    ```
+
+# **5. Experiments and methodology**
+Let us further look into our model training and evaluation methodology
+
+## **Evaluation of tfm_ner with `tiny-bert` and `bert`**
+We are evaluating `tiny-bert` and `bert-base-uncased` model on following datasets.
+
+
+## **Dataset description**
+We are using two different dataset.
+
+**1. CONLL2003 [dataset](https://huggingface.co/datasets/conll2003)** is a named entity recognition dataset released as a part of CoNLL-2003 shared task. CoNLL-2003 dataset have **4 labels**(PER, ORG, LOC, MISC).
+
+**2. FEW-NERD [dataset](https://ningding97.github.io/fewnerd/)** is a large-scale, fine-grained manually annotated named entity recognition dataset, which contains `8 coarse-grained(Major)` types, `66 fine-grained(All)` types labels.
+
+### **Dataset details**
+| Dataset Name                   | train dataset             | validation dataset         | test dataset          |
+|--------------------------------|---------------------------|----------------------------|-----------------------|
+| Conll2003	                     | 14041                     | 3250	                      | 3453                  |
+| FEW-NERD (SUP)                 | 131767                    | 18824                      | 37648                 |
+
+
+
+
+## **Training methodology**
+For training model we are using pytorch model from huggingface for token classification .
+
+* **Default training parameter are following**.
+    ```
+    "MAX_LEN": 128,
+    "TRAIN_BATCH_SIZE": 64,
+    "VALID_BATCH_SIZE": 32,
+    "EPOCHS": 50,
+    "LEARNING_RATE": 2e-05,
+    "MAX_GRAD_NORM": 10,
+    "MODE": "default"
     ```
+    ### **`Machine` description for training model**
+    * **RAM** : **`32GB`**
+    * **GPU** : **`TESLA T4`**
+    * **Memory GPU** : **`16GB`**
+
+    ### **Results**
+    ### Training on sample data from `FEW-NERD(SUP)` Dataset on `major labels`
+    | Model_Name               | Evaluation_Accuracy  | Test_Accuracy   | Test F1_Score       |Time Taken(avg)  |
+    |--------------------------|----------------------|-----------------|---------------------|-----------------|
+    |bert-base-uncased         | 0.691030008          | 0.688788764	    | 0.635250789         | 2HR+20MIN       |
```

Comment:
Let's just have 4 decimal points. So 0.6910 instead of 0.691030008.

Reply:
changes done